stylesheets/commons/Messages.css

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+Template(s): Messages
+Author(s): FO-nTTaX
+Notice: This code is deprecated and not used anymore, it has been superseeded
+by Extension:NetworkNotice and [[Special:NetworkNotice]]
```

Comment:
This file should not be added then, right?

Reply:
It is not loaded anymore, we could delete it fwiw

Reply:
Removed it and 3 additional ones, that were not set up in the ResourceLoader